backend/app/services/answer_generator.py

```
@@ -160,9 +190,83 @@ def _build_context(
+                    prompt += f"- {idx}. [{a.get('severity', 'info')}] {a.get('alert_name') or 'Unnamed alert'} at {a.get('triggered_at') or 'unknown time'} on {cam_str}: {a.get('message')}\n"  # noqa: E501
+
+                prompt += f"""
+
+CRITICAL INSTRUCTIONS:
+━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+1. Answer ONLY from the alert log data above
+2. Summarize whether any alerts fired, and roughly when and on which cameras
+3. If no alerts: clearly state that no alerts were found in the searched time window
+4. Be concise, factual, and avoid inventing alerts.
+
+Generate Natural Language Answer:"""
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)